.eslintrc

```diff
@@ -21,7 +21,7 @@
         ],
         "linebreak-style": [
             "error",
-            "unix"
+            "windows"
```

Comment:
Don't change it to windows as everyone uses Unix. You can set your linebreaks to unix in your Code Editor.